integrations/github-copilot/.port/spec.yaml

```diff
@@ -0,0 +1,22 @@
+description: github-copilot integration for Port Ocean
+icon: Cookiecutter  # Should be one of the available icons in Port
+features:
+  - type: exporter
+    section: Under Development  # Should be one of the available sections in Port
+    resources:
+      - kind: copilot-team-metrics
+      - kind: copilot-organization-metrics
+configurations:
+  - name: githubToken
+    type: string
+    sensitive: true
+    required: true
```

Comment:
description point to how generate the github token :) e.g. https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens